miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/security/DefaultAuthorizationManager.java

```diff
@@ -106,9 +111,25 @@ public void throwIfNotReadable(SecurableByProfile resource) throws IOException,
     }
   }
 
+  @Override
+  public void throwIfNotReadable(SecurableByProfile resource, User user) throws AuthorizationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'uk.ac.bbsrc.tgac.miso.service.security.AuthorizationException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)